phpcs.xml.dist

```diff
@@ -18,6 +18,9 @@
     <file>tests</file>
 
     <rule ref="Doctrine">
+        <!-- Prevents useful type assertions -->
+        <exclude name="SlevomatCodingStandard.Commenting.InlineDocCommentDeclaration.MissingVariable" />
```

Comment:
I wonder it prevents exactly 🤔

Reply:
For example an `@var` after an if condition that proves the type of a variable.